src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,123 @@ The accordion component lets users show and hide sections of related content on
 
 ## How it works
 
 There are 2 ways to use the accordion component. You can use HTML or, if you’re using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
 
 The accordion component uses JavaScript. When JavaScript is not available, users will see all the content displayed with the section labels as headings.
 
-### Find out how your users interact with the content
+### Section heading buttons
+
+An accordion will usually start with all sections hidden. To show a section, the user can interact anywhere in the heading button.
+
+The heading button includes all of these areas:
+- heading text
+- summary text (if you decide to add one)
+- call-to-action text to 'show' or 'hide'
+
+For users of screen readers, all the text in the button will be read as a single statement (separated by commas to allow for slight pauses). There’s also some visually hidden content in the heading text to help announce the call-to-action as 'show this section' or 'hide this section'
+
+{{ example({group: "components", item: "accordion", example: "default", html: true, nunjucks: true, open: false, size: "xl"}) }}
+
+#### Write clear button text
+
+Write the heading and summary text like any other button text and:
+- use sentence case
+- describe the content that will show
+- keep it short
 
-Users might need the sections they have opened to stay open if they leave and then return to the page. You can configure the accordion component to stay open.
+Users of screen readers might find it difficult to navigate the accordion if the button text is too long.
 
-Do user research to find out the best order for the sections.
+If you struggle to come up with clear button text, it might be because the way you’ve separated the content is not clear. Organise sections in a way that makes sense to users, based on their needs.
 
-### Use clear labels
+#### Adding a summary line
 
-Accordions hide content, so the labels need to be clear. If necessary, you can add a summary line to help users understand what is in the section.
+Only add a summary line if it’s actually needed, as it's likely to make the button text too long.
+
+If you’ve decided that the summary line is needed, you must make it as short as possible.
 
 {{ example({group: "components", item: "accordion", example: "with-summary-section", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
-If you struggle to come up with clear labels, it might be because the way you’ve separated the content is not clear.
+#### Structure section headings with the rest of the page
+
+The accordion component shows section headings as `<h2>` headings. If needed, change the heading level of the section headings to make them fit within the other headings on the page.
+
+### Change how an accordion starts
+
+Users might need some sections to be open from the start. They might also need sections they opened to stay open if they leave and then return to the page.
```

Comment:
In 2nd sentence, could front-load the scenario and add a 'breathing space' comma.

```suggestion
Users might need some sections to be open from the start. If they leave and then return to the page, they might also need sections they opened to stay open.
```